pkg/skuba/version.go

```diff
@@ -26,26 +26,36 @@ var (
 	Version   string
 	BuildDate string
 	Commit    string
+	Tag       string
 )
 
 type SkubaVersion struct {
 	Version   string
 	BuildType string
 	BuildDate string
 	Commit    string
+	Tag       string
 	GoVersion string
 }
 
 func CurrentVersion() SkubaVersion {
-	return SkubaVersion{
+	skubaVersion := SkubaVersion{
 		Version:   Version,
 		BuildType: BuildType,
 		BuildDate: BuildDate,
 		Commit:    Commit,
+		Tag:       Tag,
 		GoVersion: runtime.Version(),
 	}
+	if skubaVersion.Version == "" {
+		skubaVersion.Version = "untagged"
```

Comment:
should this just unset skubaVersion.Tag? I don't see this checked anywhere and the version check for untagged release is:

```	
case skubapkg.Tag == "":
    fmt.Println("** This is an UNTAGGED version and NOT intended for production usage. **")
```

Reply:
Tag will already be empty in this case; if `Version` was empty is because `Tag` was (it couldn't be inferred).